import { InvalidArgumentError } from '../error';
export { InvalidArgumentError };
export declare function repeat<T>(iterable: Iterable<T>, times: number): Iterable<T>;
